source/octf/proto/statistics.proto

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright(c) 2012-2018 Intel Corporation
+ * SPDX-License-Identifier: BSD-3-Clause-Clear
+ */
+
+syntax = "proto3";
+option cc_generic_services = true;
+import "trace.proto";
+package octf.proto;
+
+/**
+ * Distribution of values. It provides info about: maximum value, minimum value,
+ * average value, etc.
+ */
+message Distribution {
+    /**
+     * Unit for values of this distribution
+     */
+    string unit = 1;
+
+    /**
+     * Average value of distribution's items
+     */
+    uint64 avarage = 2;
+
+    /**
+     * Minimum value of distribution's items
+     */
+    uint64 min = 3;
+
+    /**
+     * Maximum value of distribution's items
+     */
+    uint64 max = 4;
+
+    /**
+     * Count of items in this distribution
+     */
+    uint64 count = 5;
+
+    /**
+     * Total sum of items in this distribution
+     */
+    uint64 total = 6;
+}
+
+/**
+ * IO statistics metric
+ */
+message IoStatisticsMetric {
+    /**
+     * Metric name
+     */
+    string name = 1;
+    /**
+     * Metric Unit
+     */
+    string unit = 2;
+    /**
+     * Metric value
+     */
+    double value = 3;
+}
+
+message IoStatisticsEntry {
+    /**
+     * Size distribution
+     */
+    Distribution size = 1;
```

Comment:
I'm thinking that it is possible we could add more distributions in the future - then maybe it's worth reserving fields  e.g. 3-20 or even making distributions a repeated field (and adding a name field in Distrubtion)